ARTIFACTS: unbound variable error when running make test-e2e

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://kubernetes.io/security/
-->

**What happened**:
When running make test-e2e, encountered the following error: 

```
./hack/e2e-test.sh: line 46: ARTIFACTS: unbound variable
Deleting cluster "kind" ...
Deleted nodes: ["kind-control-plane"]
make: *** [Makefile:137: test-e2e] Error 1
```

**What you expected to happen**:
For e2e tests to run

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:

- Kubernetes version (use `kubectl version`):
- LWS version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -11,9 +11,6 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-GO_CMD ?= go
-
-
 # CONTAINER_TOOL defines the container tool to be used for building images.
 # Be aware that the target commands are only tested with Docker which is
 # scaffolded by default. However, you might want to replace it to use other
@@ -150,7 +147,7 @@ verify: lint toc-verify
 
 .PHONY: build
 build: manifests fmt vet ## Build manager binary.
-	$(GO_BUILD_ENV) $(GO_CMD) build -o bin/manager cmd/main.go
+	go build -o bin/manager cmd/main.go
 
 .PHONY: run
 run: manifests fmt vet ## Run a controller from your host.

